.travis.yml

```diff
@@ -31,8 +31,9 @@ before_install:
   - sudo apt-get -q update
 
 install:
-  - sudo apt-get install -y python
+  - sudo apt-get install -y python python3 python3-pip
   - pip install grpcio scapy codecov
+  - pip3 install grpcio coverage
```

Comment:
Also we would need to replace `str(scapy packet)` with `bytes(scapy packets)` in BESS scripts... The scapy3k webpage says:

> N.B.! As a difference from scapy for python2, use bytes() instead of str() when converting packet to bytes. Also, most arguments expect bytes value instead of str value except the ones, which are naturally suited for human input (e.g. domain name).